draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1224,26 +1224,30 @@ The following types of unlinkability are considered here:
 
 * Presentation Unlinkability: A Verifier should not be able to link two
   presentations of the same credential.
-* Verifier/Verifier Unlinkability: Two colluding Verifiers should not be able to
-  learn that they have received presentations of the same credential.
+* Verifier/Verifier Unlinkability: The presentations made to two different
+  Verifiers should not reveal that the same credential was presented (e.g., if the two
+  Verifiers collude, data leaks from one Verifier to the other, or if both
+  are forced by a third party to reveal the presentations made to them).
 * Issuer/Verifier Unlinkability (Honest Verifier): An Issuer of a credential
-  should not be able to know that a user presented the credential to a certain
-  Verifier that is not behaving maliciously.
-* Issuer/Verifier Unlinkability (Colluding/Compromised Verifier): An Issuer of a
-  credential should not be able to tell that a user presented the credential to
-  a certain Verifier, even if the Verifier colludes with the Issuer or becomes
-  compromised and leaks stored credentials from presentations.
+  should not be able to know that a user presented this credential if
+  the Verifier is not sharing presentation data with the Issuer
+  accidentally, deliberately, or because it is forced to do so.
+* Issuer/Verifier Unlinkability (Colluding/Compromised/Coerced Verifier): An Issuer of a
+  credential should not be able to tell that a user presented this credential to
+  a certain Verifier, even if the Verifier accidentally or deliberately shares
+  presentation data with the Issuer or is forced to do so.
```

Comment:
```suggestion
  should not be able to know that a user presented this credential unless
  the Verifier is sharing presentation data with the Issuer
  accidentally, deliberately, or because it is forced to do so.
* Issuer/Verifier Unlinkability (Colluding/Compromised/Coerced Verifier): An Issuer of a
  credential should under no circumstances be able to tell that a user presented this credential to
  a certain Verifier. In particular this includes cases when the Verifier accidentally or deliberately shares
  presentation data with the Issuer or is forced to do so.
```